onnx/defs/__init__.py

```diff
@@ -6,38 +6,21 @@
 import onnx
 import onnx.onnx_cpp2py_export.defs as C
 
-
 ONNX_DOMAIN = ""
 
+has = C.has_schema
+get_schema = C.get_schema
+get_all_schemas = C.get_all_schemas
+get_all_schemas_with_history = C.get_all_schemas_with_history
 
-def has(op_type):
-    return C.has_schema(op_type)
-
-
-def onnx_opset_version():
-    return C.schema_version_map()[ONNX_DOMAIN][1]
-
-
-def get_schema(op_type, max_inclusive_version=onnx_opset_version()):
-    return C.get_schema(op_type, max_inclusive_version)
```

Comment:
@raymondxyang 
You've removed default value of max_inclusive_version and get that error.

Reply:
let's add the default argument in the pybind11 bindings

Reply:
Gonna revert this first until we figure out on which step(s) we need to define a default value.. currently we have: 1) py apis(some with default value) 2) pybind(no default value) 3) c apis(default value defined)

Reply:
Both 2 and 3 don't have default value I think.
And we should not add default value to schema.h because of there is another constructor which is using maxInclusiveVersion.
What we need to do I think is just add default value to pybind11 bindings as @bddppq said.
Maybe we can also add default value of domain.